lib/govuk_app_config/configure.rb

```diff
@@ -42,3 +42,14 @@
     GovukStatsd.increment("error_reports_failed")
   }
 end
+
+Rails.application.config.xray = {
+  name: ENV["GOVUK_APP_NAME"].to_s,
+  patch: %I[net_http aws_sdk],
+  sampling_rules: {
+    default: {
+      "fixed_target": 0,
+      "rate": 0.01,
```

Comment:
Might be nice to make this configurable, but with a default of `0.01`.

Reply:
Done